<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Apropos</class>
 <widget class="QMainWindow" name="Apropos">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>510</width>
    <height>271</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <widget class="QGraphicsView" name="imagePreviSat">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>221</width>
      <height>221</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="nomLogiciel">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>10</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string notr="true">PreviSat 3.0</string>
    </property>
   </widget>
   <widget class="QLabel" name="numeroVersion">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>30</y>
      <width>161</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Version 3.0.0.0 (20 mars 2012)</string>
    </property>
   </widget>
   <widget class="QLabel" name="informationsCopyright">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>50</y>
      <width>181</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string notr="true">Copyright © Astropedia 2005-2012</string>
    </property>
   </widget>
   <widget class="QLabel" name="siteInternet">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>70</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string notr="true">&lt;a href=&quot;http://astropedia.free.fr/&quot;&gt;http://astropedia.free.fr&lt;/a&gt;</string>
    </property>
    <property name="openExternalLinks">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="plainTextEdit">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>90</y>
      <width>251</width>
      <height>141</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="plainText">
     <string>PreviSat est un logiciel de calcul de la position des satellites artificiels.
Il permet d'afficher la position des satellites à tout instant (en temps réel ou en mode manuel), d'effectuer les prévisions de passage pour n'importe quel lieu d'observation, et également de déterminer très rapidement les flashs Iridium.

PreviSat est entièrement gratuit!</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ok">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>240</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>OK</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
